### PR TITLE
SDN-4154:Add troubleshooting section for upgrades to OVN IC

### PR DESCRIPTION
**Pull request by ricky-rav**
- Add section for troubleshooting upgrades to OVN-IC, now possible thanks to https://github.com/openshift/enhancements/pull/1567/commits/b9d6c0d55e2391f8b453172aae5a7c048f8e840e
- updated number of ovnkube-control-plane pods to 2, because of https://github.com/openshift/enhancements/pull/1567/commits/b5d5caf076f1d2ef187d02dc92d9605af2b0aba5

### CODE REVIEW

File: enhancements/network/upgrades-with-ovn-ic.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ data:
   zone-mode: singlezone
   temporary: true
 ```
+
+The "temporary: true" field above is only used up until 4.14.13 and is omitted in later 4.14.z.
```
Comment (tssurya) [inline enhancements/network/upgrades-with-ovn-ic.md:105]: hmm should we stretch this a bit further and make it "omitted in releases >= 4.14.14 and >= 4.15 ?" I don't know if talking specifically about 4.14 z stream implies that 4.15 and other minor versions are not even effected by this..same comment here and elsewhere you mention versioning..
or maybe in context of upgrades 4.13 -> 4.15 will always go via 4.14 so we are ok here?

File: enhancements/network/upgrades-with-ovn-ic.md
Comment (tssurya) [inline enhancements/network/upgrades-with-ovn-ic.md:1]: nit: wanna change "last updated" ?

File: enhancements/network/upgrades-with-ovn-ic.md (hunk 1 above)
Comment (ricky-rav) [reply enhancements/network/upgrades-with-ovn-ic.md:105]: It only ever applies to upgrades to 4.14.z. There's no skipping releases when you upgrade :)

**Comment by CFields651** [comment]
@ricky-rav the diagrams an doc are awesome!  I think we could use a bit more explanation around _"If ever a cluster is not making progress during the upgrade to OVN IC..."_  Like why would an upgrade not or stop making progress?  What is lost by skipping phase 1 or phase 2; is there anything that needs to be cleaned up?

**Comment by CFields651** [comment]
LGTM!

**Review by trozet — APPROVED** [review]
/lgtm

thanks @ricky-rav

**Pull request merged**